packages/webpack-cli/lib/groups/HelpGroup.js

```diff
@@ -39,7 +40,30 @@ class HelpGroup {
         process.stdout.write('\n                  Made with ♥️  by the webpack team \n');
     }
 
-    outputVersion() {
+    outputVersion(externalPkg) {
+        const commandsUsed = (commands) => {
+            return process.argv.filter((val) => commands.find(({ name }) => name === val));
+        };
+
+        if (externalPkg && commandsUsed(commands).length === 1) {
+            try {
+                if (commands.filter((cmd) => cmd.name === externalPkg.name)) {
+                    const { name, version } = require(`@webpack-cli/${externalPkg.name}/package.json`);
+                    process.stdout.write(`\n${name} ${version}`);
+                } else {
+                    const { name, version } = require(`${externalPkg.name}/package.json`);
+                    process.stdout.write(`\n${name} ${version}`);
+                }
+            } catch (e) {
+                console.error(chalk.red('\nError: External package not found.'));
+                process.exitCode = -1;
+            }
+        }
+
+        if (commandsUsed(commands).length > 1) {
+            throw new Error('You provided multiple commands. Please use only one command at a time.');
```

Comment:
It would be better not to throw and rather log the respective message as from the `catch` block above.

Reply:
this will still output the version of `webpack` & `webpack-cli`.

```
You provided multiple commands. Please use only one command at a time.

webpack-cli 4.0.0-beta.8
webpack 4.42.1
```

but I think in case of multiple commands we should exit the process. What do you say?

Reply:
Ig there is no harm in having the version information (`webpack` & `webpack-cli`) to show up along with the respective warning message. Or else, a forceful exit would do.
Well, let's wait to hear out from other maintainers.